shardingsphere-features/shardingsphere-ha/shardingsphere-ha-provider/shardingsphere-ha-mgr/pom.xml

```diff
@@ -22,7 +22,7 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.apache.shardingsphere</groupId>
-        <artifactId>shardingsphere-ha</artifactId>
+        <artifactId>shardingsphere-ha-provider</artifactId>
```

Comment:
What is the proposal here?

Reply:
> What is the proposal here?

When I use maven clean intall all projects, it has a error at here. And this project is the child of shardingsphere-ha-provider. So I modify it and install again, thers is no more errors.